### ISSUE TITLE
antrax graph-explore refuses to work with MCR

### ISSUE THREAD

**Issue by janamach** [Social-Evolution-and-Behavior/anTraX#6]
Long story short, `antrax graph-explorer` does not work for me on Ubuntu 20.04. I tried with both MCR and MATLAB installation, the end result is the same (the GUI doesn't load), but they behave differently.

With MCR, it exits nicely without attempting to load the GUI:

```
(antrax-debug-jan21) jana@str09:~/src/anTraX-data$ antrax graph-explorer JS16/


==================================================================================

Welcome to anTraX - a software for tracking color tagged ants (and other insects)

==================================================================================

02/02/21 17:21:37 -D- running matlab mcr 
02/02/21 17:21:37 -D- command is: /home/jana/src/anTraX/bin/antrax_glnxa64_mcr_interface graph_explorer_app JS16/ m 0
02/02/21 17:21:44 -D- matlab app exited with code 0
(antrax-debug-jan21) jana@str09:~/src/anTraX-data$
```

With MATLAB (ran remotely, see in issue #5 why) it does load a GUI, but it looks like this:

![2021-02-02_17-27-12](https://user-images.githubusercontent.com/39128466/106630339-08a03800-657c-11eb-899d-2e819ef104b2.png)

All other GUIs seem to load fine. Any ideas what could be the issue?

**Comment by asafgal** [comment]
Hey Jana,

The gui with matlab actually looks fine, just choose a specific video in the "Graph" menu, and then enter the time slice to plot. I probably need to make this gui a bit more friendly :-/

Regarding the mcr version, not sure what is going on. Can you try giving it an explicit video on the command line?
```
antrax graph-explorer JS16 --m 1
```

**Comment by janamach** [comment]
Hi Asaf,

Thank you for responding so quickly! 

>The gui with matlab actually looks fine, just choose a specific video in the "Graph" menu

Ah, my bad! It does work (just checked it). Sorry for not trying it immediately, my expectations were clearly biased towards a negative outcome :-)

With MCR, specifying the video in the command line did not help:

```
(antrax-debug-jan21) jana@str09:~/src/anTraX-data$ antrax graph-explorer JS16 -m 1

==================================================================================

Welcome to anTraX - a software for tracking color tagged ants (and other insects)

==================================================================================

02/02/21 18:06:05 -D- running matlab mcr 
02/02/21 18:06:05 -D- command is: /home/jana/src/anTraX/bin/antrax_glnxa64_mcr_interface graph_explorer_app JS16 m 1
02/02/21 18:06:15 -D- matlab app exited with code 0

```

**Comment by asafgal** [comment]
ok, I found a small bug in the launching code. Pull (the jana-debug branch, I haven't merged yet) and try again

**Comment by janamach** [comment]
It helped! It loads successfully on my Debian 10 laptop at home, I expect the Ubuntu 20.04 computer at work to behave the same. The output I get looks like this now:

```
(antrax-debug-feb2) jana@corvus:~/src/anTraX-data$ antrax graph-explorer JS16/

==================================================================================

Welcome to anTraX - a software for tracking color tagged ants (and other insects)

==================================================================================

02/02/21 22:01:46 -D- running matlab mcr 
02/02/21 22:01:46 -D- command is: /home/jana/src/anTraX/bin/antrax_glnxa64_mcr_interface graph_explorer JS16/ m 0
22:01:57 -D- initializing expreader object
22:01:57 -I- Reading video information from file
22:02:01 -D- experiment loaded
22:02:01 -I- Ready!
22:02:19 -I- Loading trgraph from antrax_demo/graphs/graph_4_4.mat
22:02:20 -I- Finished loading trgraph with 219 tracklets
22:02:20 -D- graph loaded
22:02:27 -I- Loading trgraph from antrax_demo/graphs/graph_6_6.mat
22:02:27 -I- Finished loading trgraph with 172 tracklets
22:02:28 -D- graph loaded
22:02:36 -I- Loading trgraph from antrax_demo/graphs/graph_2_2.mat
22:02:37 -I- Finished loading trgraph with 251 tracklets
22:02:37 -D- graph loaded
22:02:42 -I- Loading trgraph from antrax_demo/graphs/graph_1_1.mat
22:02:42 -I- Finished loading trgraph with 202 tracklets
22:02:42 -D- graph loaded
02/02/21 22:02:59 -D- matlab app exited with code 0
(antrax-debug-feb2) jana@corvus:~/src/anTraX-data$
```

Thank you :-)

**Comment by asafgal** [comment]
Fixed in commit b3def69f1dd3fe76f007f50d4221f45bb0662456
